node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -301,6 +313,49 @@ pub fn configure_database(
     Ok(())
 }
 
+fn set_neighbors_in_zero_hop_if_supplied(
+    multi_config: &MultiConfig,
+    neighborhood_mode: NeighborhoodModeLight,
+    persistent_config: &mut dyn PersistentConfiguration,
+) -> Result<(), ConfiguratorError> {
+    fn split_descriptors(whole_input: String) -> Vec<NodeDescriptor> {
```

Comment:
Is this already implemented somewhere else? It seems like it ought to be. Can we use that implementation here?